code/modules/antagonists/traitor/objectives/steal.dm

```diff
@@ -0,0 +1,236 @@
+	/// A list of 2 elements, which contain the range that the time will be in. Represented in minutes.
+	var/hold_time_required = list(5, 15)
+	/// The current time fulfilled around the item
+	var/time_fulfilled = 0
+	/// The maximum distance between the bug and the objective taker for time to count as fulfilled
+	var/max_distance = 4
+	/// The bug that will be put onto the item
+	var/obj/item/traitor_bug/bug
+	/// Any special equipment that may be needed
+	var/list/special_equipment
+
+	abstract_type = /datum/traitor_objective/steal_item
+
+/datum/traitor_objective/steal_item/low_risk_cap
+	progression_minimum = 5 MINUTES
+	progression_maximum = 20 MINUTES
+
+	progression_reward = list(5 MINUTES, 10 MINUTES)
+	telecrystal_reward = 1
+	possible_items = list(
+		/datum/objective_item/steal/low_risk/techboard/borgupload,
+		/datum/objective_item/steal/low_risk/techboard/aiupload,
+		/datum/objective_item/steal/low_risk/aicard,
+	)
+
+/datum/traitor_objective/steal_item/low_risk
+	progression_minimum = 10 MINUTES
+	progression_reward = list(5 MINUTES, 10 MINUTES)
+	telecrystal_reward = 1
+
+	possible_items = list(
+		/datum/objective_item/steal/low_risk/bartender_shotgun,
+		/datum/objective_item/steal/low_risk/fireaxe,
+		/datum/objective_item/steal/low_risk/nullrod,
+		/datum/objective_item/steal/low_risk/cargo_budget,
+		/datum/objective_item/steal/low_risk/clown_shoes,
+	)
+
+/datum/traitor_objective/steal_item/somewhat_risky
+	progression_minimum = 20 MINUTES
+	progression_reward = 5 MINUTES
+	telecrystal_reward = list(1, 2)
+
+	possible_items = list(
+		/datum/objective_item/steal/magboots,
+		/datum/objective_item/steal/hypo,
+		/datum/objective_item/steal/reactive,
+		/datum/objective_item/steal/handtele,
+		/datum/objective_item/steal/blueprints,
+	)
+
+/datum/traitor_objective/steal_item/somewhat_risky/generate_objective(datum/mind/generating_for, list/possible_duplicates)
+	if(!handler.get_completion_count(/datum/traitor_objective/steal_item/low_risk))
+		return FALSE
+	return ..()
+
+/datum/traitor_objective/steal_item/risky
+	progression_minimum = 30 MINUTES
+	progression_reward = 5 MINUTES
+	telecrystal_reward = list(2, 4)
+
+	possible_items = list(
+		/datum/objective_item/steal/reflector,
+		/datum/objective_item/steal/capmedal,
+		/datum/objective_item/steal/blackbox,
+		/datum/objective_item/steal/hdd_extraction,
+		/datum/objective_item/steal/documents,
+	)
+
+/datum/traitor_objective/steal_item/risky/generate_objective(datum/mind/generating_for, list/possible_duplicates)
+	if(!handler.get_completion_count(/datum/traitor_objective/steal_item/somewhat_risky))
+		return FALSE
+	return ..()
+
+/datum/traitor_objective/steal_item/very_risky
+	progression_minimum = 40 MINUTES
+	progression_reward = 8 MINUTES
+	telecrystal_reward = list(3, 6)
+
+	possible_items = list(
+		/datum/objective_item/steal/hoslaser,
+		/datum/objective_item/steal/caplaser,
+		/datum/objective_item/steal/nuke_core,
+		/datum/objective_item/steal/supermatter,
```

Comment:
Is this specifically holding the sliver, or the loaded sealed container? Because you cannot touch the sliver, period, it dusts objects and people that touch it (and then deletes).